src/v/pandaproxy/json/requests/fetch.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "bytes/iobuf.h"
+#include "bytes/iobuf_parser.h"
+#include "json/json.h"
+#include "kafka/errors.h"
+#include "kafka/requests/fetch_request.h"
+#include "model/record.h"
+#include "model/record_batch_reader.h"
+#include "pandaproxy/json/iobuf.h"
+#include "pandaproxy/json/requests/error_reply.h"
+#include "pandaproxy/json/requests/produce.h"
+#include "pandaproxy/json/types.h"
+#include "pandaproxy/types.h"
+#include "seastarx.h"
+
+#include <seastar/core/sstring.hh>
+
+#include <rapidjson/reader.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
+namespace pandaproxy::json {
+
+template<>
+class rjson_serialize_impl<kafka::fetch_response::partition> {
```

Comment:
would it be reasonable to split this out into a separate commit like `class rjson_serialize_impl<iobuf>` ?

Reply:
Yes, and I seem to have dropped my tests somewhere.